src/Tizen.NUI/src/public/BaseComponents/ViewPublicMethods.cs

```diff
@@ -303,7 +303,7 @@ public void ClearBackground()
         /// <since_tizen> 3 </since_tizen>
         public void Show()
         {
-            if (Accessibility.Accessibility.Enabled && ((GetAccessibilityStates() & AccessibilityStates.Modal) != 0))
+            if ((GetAccessibilityStates() & AccessibilityStates.Modal) != 0)
```

Comment:
The Accessibility.Accessibility.Enabled condition has to be removed in this place, as  the existence of any Modal View has to be registered even if Accessibility mode is off, otherwise when Screen Reader is being turned on when the Menu or any other popup-like View is currently shown, the UI context root will not be properly set and the user may navigate out of the context.

Reply:
This issue needs more investigation, so I removed this change from the current commit.